deps/key-value-storage/src/lib.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2025 by Alibaba.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! A key-value storage interface and implementations.
+
+use async_trait::async_trait;
+use serde::Deserialize;
+
+use std::{sync::Arc, time::Duration};
+
+pub mod error;
+pub use error::{KeyValueStorageError, Result};
+
+pub mod simple;
+
+#[cfg(feature = "postgres")]
+pub mod postgres;
+
+#[derive(Default)]
+pub struct SetContext {
```

Comment:
I think "Context" could be misleading here. I would suggest "SetParams", "Context" usually provides meta data and hooks that are independent of the business logic of a fn (like correlation_id, UserId)

Reply:
Nice name suggestion. Renamed to `SetParameters`